Library/Formula/ffmpeg.rb

```diff
@@ -32,6 +32,7 @@ class Ffmpeg < Formula
   option "with-x265", "Enable x265 encoder"
   option "with-libsoxr", "Enable the soxr resample library"
   option "with-webp", "Enable using libwebp to encode WEBP images"
+  option "with-zmq", "Enable using libzmq to receive commands sent through a libzmq client"
```

Comment:
This will need to be changed as well to `zeromq`, otherwise a user is going to have to pass `brew install ffmpeg --with-zeromq --with-zmq` to `brew` to install with the desired option.